ci/test_integration.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Call this script with name of container as parameter
+# [merlin-training, merlin-tensorflow-training,
+#  merlin-pytorch-training, merlin-inference]
+
+# Get last NVTabular version
+cd /nvtabular/
+git pull origin main
+
+# Run tests for all containers
+#pytest tests/integration/test_notebooks.py::test_criteo_example
+pytest tests/integration/test_notebooks.py::test_rossman_example
+pytest tests/integration/test_notebooks.py::test_movielens_example
+
+# Run tests for specific containers
+if [ "$1" == "merlin-training" ]; then
+  pytest tests/integration/test_nvt_hugectr_training.py::test_nvt_hugectr_training
+elif [ "$1" == "merlin-tensorflow-trainig" ]; then
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_rossmann_inference
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_movielens_inference
+elif [ "$1" == "merlin-pytorch-training" ]; then
+  echo "Nothing specific for $1 yet"
+elif [ "$1" == "merlin-inference" ]; then
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_rossmann_inference_triton
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_rossmann_inference_triton_mt
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_movielens_inference_triton
+  pytest tests/integration/test_nvt_tf_inference.py::test_nvt_tf_movielens_inference_triton_mt
+  pytest tests/integration/test_nvt_hugectr_inference.py::test_nvt_hugectr_inference
+else
+  echo "INVALID Container name"
+  exit 1
+fi
+
```

Comment:
So, for nvt + tf inference integration tests, you first run the line 13 and 14, then 20 and 21, and lastly 25, 26, 27, and 28, right? If that's the case,

Reply:
Correct. 

The idea is to run first the integration tests for merlin-training, merlin-tensorflow-training, and merlin-pytorch-training; and when it is done, run for merlin-inference